meta-leda-distro/recipes-core/base-files/base-files/qemuarm64/fstab

```diff
@@ -7,6 +7,7 @@ tmpfs                /run                 tmpfs      mode=0755,nodev,nosuid,stri
 tmpfs                /var/volatile        tmpfs      defaults              0  0
 
 # Add mount for boot, grubenv and data partition
-LABEL=boot           /boot                auto       defaults,nofail,noatime  0  0
+/dev/vda1           /boot                vfat       defaults,nofail,noatime  0  0
```

Comment:
A shot in the blue: try "LABEL=BOOT" (uppercase). It may be that on a MSDOS/vfat partitions, the label gets automatically converted to uppercase, since MSDOS filesystems didnt support lower case. 1980 technology greets.

Reply:
Used uppercase BOOT, thanks for the suggestion!